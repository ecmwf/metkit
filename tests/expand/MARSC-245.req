retrieve,
    DOMAIN=EUROATL,LEVTYPE=110,LEVELIST=1,DATE=20040102,TIME=1800,STEP=6,PARAM=33.201,CLASS=MS,TYPE=FG,STREAM=OPER,EXPVER=0001,COUNTRY=IT,MODEL=HRM,BCMODEL=IFS,ICMODEL=MVOI,
    target=reference.data 