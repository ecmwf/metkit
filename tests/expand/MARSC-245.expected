RETRIEVE,
    CLASS      = MS,
    COUNTRY    = IT,
    MODEL      = HRM,
    BCMODEL    = IFS,
    ICMODEL    = MVOI,
    DOMAIN     = EUROATL,
    STREAM     = OPER,
    TYPE       = FG,
    EXPVER     = 0001,
    LEVTYPE    = 110,
    PARAM      = 201033,
    TIME       = 1800,
    DATE       = 20040102,
    STEP       = 6,
    LEVELIST   = 1,
    TARGET     = reference.data,
    GRIB       = ECMWF